### DISCUSSION TITLE
b-form-select is not working for html fields

### DISCUSSION THREAD

**Discussion by debamishra14** [bootstrap-vue/bootstrap-vue#5988] [Q&A] [answered]
I am trying to use some HTML styling in b-form-select, but unfortunately, that is not working.
It is mentioned to do as below and I tried that but no success. Can anyone suggest how to fix it?
```
const options = [
  { text: 'Item 1', value: 'first' },
  { text: 'Item 2', value: 'second' },
  { html: '<b>Item</b> 3', value: 'third', disabled: true },
  { text: 'Item 4' },
  { text: 'Item 5', value: { foo: 'bar', baz: true } }
]
```

code pen: https://codepen.io/kkdas202/pen/YzWajoJ?editors=1010

**Comment by Hiws** [comment, marked as answer]
If you look in the DOM you should see that the tags are actually rendered, so it's working as intended.
The issue is that most browsers don't support tags inside `<option>` tags, or styling on other than `color` and `background-color`.

I do question why this option is even available in the first place, since it seems fairly useless given the limitations of HTML in `<option>` tags.

**Comment by debamishra14** [comment]
thanks, @Hiws for your clarification but is there any way we can give color? I can see `color` is also not getting applied.

**Comment by Hiws** [comment]
IIRC `color` and `background-color` only works if applied directly to the `<option>`.

**Comment by debamishra14** [comment]
I want some of my test to be in blue and some are in red. Can we do it? I
want to do it in one option

Like <option><red text><blue text></option>

**Comment by Hiws** [comment]
I don't think that's possible.
